integration_tests/dbt_project.yml

```diff
@@ -1,22 +1,22 @@
 name: 'salesforce_integration_tests'
-version: '1.1.1'
+version: '1.2.0'
 config-version: 2
 
 profile: 'integration_tests'
 
 models:
   +materialized: table
-  +schema: "salesforce_{{ var('directed_schema','dev') }}"
+  # +schema: "salesforce_{{ var('directed_schema','dev') }}" --Used for validation tests
```

Comment:
This doesn't need to be commented out. Please comment back in.

Reply:
Uncommented.